dotCMS/src/main/java/com/dotcms/rest/api/v1/maintenance/MaintenanceResource.java

```diff
@@ -65,15 +68,22 @@ public final Response shutdown(@Context final HttpServletRequest request,
                         .rejectWhenNoUser(true).requiredPortlet("maintenance").init();
 
 
-        Logger.info(this.getClass(), "User:" + initData.getUser() + " is shutting down dotCMS!");
+        Logger.info(this.getClass(), "User:" + initData.getUser() + " is shutting down dotCMS!"); 
         SecurityLogger.logInfo(this.getClass(),
                         "User:" + initData.getUser() + " is shutting down dotCMS from ip:" + request.getRemoteAddr());
 
         if (!Config.getBooleanProperty("ALLOW_DOTCMS_SHUTDOWN_FROM_CONSOLE", true)) {
             return Response.status(Status.FORBIDDEN).build();
         }
 
-        Runtime.getRuntime().exit(0);
+        DotConcurrentFactory.getInstance()
+                .getSubmitter()
+                .submit(
+                        () -> Runtime.getRuntime().exit(0),
```

Comment:
Codacy found an issue: [System.exit() should not be used in J2EE/JEE apps](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6380610)